config/zowe-api-dev/alternative_start.sh

```diff
@@ -106,12 +108,16 @@ _BPX_JOBNAME=${ZOWE_PREFIX}${DISCOVERABLECLIENT_CODE} java -Xms32m -Xmx256m -Xqu
     -Dserver.ssl.keyPassword="${KEYSTORE_PASSWORD}" \
     -Dserver.ssl.keyStore="${KEYSTORE}" \
     -Dserver.ssl.keyStorePassword="${KEYSTORE_PASSWORD}" \
+    -Dserver.ssl.keyStoreType="${KEYSTORE_TYPE}" \
```

Comment:
Does it make sense to provide a default value PKCS12?

Reply:
sure we can, I just didn't want to spend too much time on updating this one, not sure how many people use it

Reply:
done